### PR TITLE
Grocery List

### PR DESCRIPTION
**Pull request by indirap**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -137,6 +137,8 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-vector-icons')
+    compile project(':react-native-vector-icons')
```
Comment (vatthikorn) [inline android/app/build.gradle:141]: Looks like duplicate line

File: android/app/src/main/java/com/app/MainApplication.java [hunk 2]
```diff
@@ -3,6 +3,8 @@
 import android.app.Application;
 
 import com.facebook.react.ReactApplication;
+import com.oblador.vectoricons.VectorIconsPackage;
```
Comment (vatthikorn) [inline android/app/src/main/java/com/app/MainApplication.java:6]: Duplicate

File: android/app/src/main/java/com/app/MainApplication.java [hunk 3]
```diff
@@ -22,7 +24,9 @@ public boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage()
+          new MainReactPackage(),
+            new VectorIconsPackage(),
```
Comment (vatthikorn) [inline android/app/src/main/java/com/app/MainApplication.java:28]: Duplicate

File: android/settings.gradle [hunk 4]
```diff
@@ -1,3 +1,7 @@
 rootProject.name = 'app'
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```
Comment (vatthikorn) [inline android/settings.gradle:5]: Duplicate lines

File: ios/app.xcodeproj/xcuserdata/indirapranabudi.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+			<integer>1</integer>
+		</dict>
+		<key>app.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>0</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
```
Comment (vatthikorn) [inline ios/app.xcodeproj/xcuserdata/indirapranabudi.xcuserdatad/xcschemes/xcschememanagement.plist:19]: This file should have been ignored...

**Review by vatthikorn — CHANGES_REQUESTED** [review]

File: android/app/build.gradle (hunk 1 above)
Comment (indirap) [reply android/app/build.gradle:141]: Wait I don't remember changing these or any of the other Android and iOS files...

**Comment by indirap** [comment]
Closing this in favor of https://github.com/Grocee/grocee-ui/pull/4

**Pull request closed without merging**